cocos/core/components/model-renderer.ts

```diff
@@ -49,9 +49,23 @@ export class ModelRenderer extends Renderer {
         this._onVisibilityChange(val);
     }
 
+    /**
+     * @en The transparent sorting priority which will be applied to the committed models.
+     * @zh 应用于所有提交渲染的 Model 在透明队列中的排序优先级
+     */
+    get transparentPriority () {
+        return this._transparentPriority;
+    }
+
+    set transparentPriority (val) {
```

Comment:
What does this function used for? When need to invoke this function?

Reply:
When the user wants to control the priority directly